data/index.html

```diff
@@ -133,40 +133,52 @@ <h5 class="text-center">USGS Topo Maps</h5>
-    </div>
     <div class="grid__col section">
-        <h2>The Open SGID</h2>
-        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing the datasets from the SGID which are also available from the <a href="http://opendata.gis.utah.gov/">AGRC Open Data</a>. It is a publicly-available, direct database connection that can be utilized on a multitude of GIS platforms to access Utah spatial data. A <a href="https://github.com/agrc/open-sgid">GitHub repository</a> is used as the primary source of information for the Open SGID, and to catalog and address issues with explanations and or troubleshooting techniques. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
-        <p class="section-read-more">
-            <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get connected</a>
+        <h2>Welcome to the State Geographic Information Database (SGID)</h2>
+        <p class="readable">The <a href="{% link sgid/index.html %}">SGID</a> aims to be a centralized collection of GIS datasets created or maintained by state agencies. SGID datasets live in many different places depending on their type, availability, and popularity:
+        <ul class="readable"> 
+            <li>The AGRC-curated Open SGID database</li>
+            <li>ArcGIS Online hosted feature services</li>
+            <li>SGID Open Data site</li>
+            <li>Raster Discovery web app</li>
+            <li>Discover basemap and imagery services</li>
+            <li>The AGRC Web API</li>
+        </ul>
         </p>
-    </div>
-    <div class="grid__col section alternate">
-    <h2>SGID Index</h2>
-        <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:568px">
-            <a href="{% link data/sgid-index/index.html %}">
-                <img alt="SGID Index Table Search" class="text-center" src="{% link images/sgid-index.png %}">
-            </a>
-        </div>
-        <p class="readable">The SGID Index is a sortable and filterable table containing data items contained in our SGID. It is a great resource to quickly find GIS data and resources important to you. In the SGID Index, you will find the ISO category, the name of the resource, the agency who is the data steward, a short description and a link to a service endpoint if one is available. </p>
+
+        <p class="readable">The <a href="{% link data/sgid-index/index.html %}">SGID Index</a> is the master data index, regardless of where the data are stored or how they are served. It includes links to AGRC-maintained datasets as well as links to the authoritative sources for datasets created and served by other state agencies. Clicking on the <a href="#data-categories">category icons</a> on this page will take you to the associated category in the Index.</p>
```

Comment:
**issue**: this takes you to the data page index. not the aforementioned sgid index.

> Clicking on the <a href="#data-categories">category icons</a> on this page will take you to the associated category in the Index.</p>

Reply:
Are you saying that the link doesn't take you to the index? I meant for it to just jump you to the category icons themselves (otherwise this mention of "category icons" seemed too far removed from the icons themselves to be intuitive). I've edited the text a little to hopefully make this clearer. Let me know what you think.

Reply:
This link does go to the sgid index

> The <a href="{% link data/sgid-index/index.html %}">SGID Index</a>

If you click on 

> <a href="#data-categories">category icons</a>

yes, that takes you to the category icons

Once you click on an icon you go to a data page index (data/category/index.md) not the sgid index (data/sgid-index/index.html).

It makes me think you are referring to the website or the data/categories/index.md as the sgid index when the sgid index is only the single page on the website.

Reply:
Ok, yeah, I see what you're saying now. You're right, I was assuming that the "SGID Index" referred to all the information we have on the website about SGID datasets, and data/sgid-index was just a searchable index page. 

That being the case, none of these edits refer to the data pages directly. Do we need to introduce them differently, or is it a matter of semantics and we just start referring to the data pages as the SGID index?

I think this gets back to the big picture question of "What is the SGID?", and it seems like we've gone towards the direction that the SGID is a collection of Utah GIS datasets. Some, the AGRC-curated datasets, are hosted by us and either maintained directly or in partnership with other agencies.  Others are not maintained or hosted by us, so we just provide references, in the form of links on the data pages/index or through the Esri-land Open Data site.

Reply:
> Do we need to introduce the data pages differently

I don't think so. I think we can clarify that the index will take you to a data page that could have supplementary metadata or downloads not able to be stored in open data/arcgis online

> is it a matter of semantics and we just start referring to the data pages as the SGID index?

I've always thought of the data pages as metadata and the index as, well, the index to find a data page.

Since not every item in what I call the sgid index has a data page I would like to keep their naming separate. But you do bring up the point that we don't discuss the data pages at all.